### PR TITLE
add src file to npm package

### LINKED ISSUE #273: Gatsby try to load gatsby-node.js source in a non-existent path
```log
UNHANDLED REJECTION ENOENT: 
no such file or directory, open '/node_modules/gatsby-plugin-robots-txt/src/gatsby-node.js'
```

Gatsby didn't load root directory of plugin `gatsby-plugin-robots-txt/gatsby-node.js`

I've try two gatsby-plugin-robots-txt version: 1.5.1, 1.5.0

Gatsby CLI version: 2.12.34
Gatsby version: 2.22.9

### PR DESCRIPTION
**Pull request by b1551**
Fixes #273 
Source file is missing in NPM package which causes as error since the file is referenced in gatsby-node.js.map file

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,5 +50,6 @@
   },
   "jest": {
     "collectCoverage": true
-  }
+  },
+  "version": "1.5.2"
```
Comment (mdreizin) [inline package.json:54]: Please remove `version`, because it will be added automatically.

Comment (b1551) [reply package.json]: Done

**Comment by mdreizin** [comment]
@b1551 Could you please enable GitHub Actions on your fork?

**Comment by b1551** [comment]
Hi @mdreizin , GitHub Actions are enabled

**Comment by mdreizin** [comment]
@b1551 I am sorry for asking to enable GitHub Actions on a for. I was wrong, just merged a fix to have working GitHub workflow.

**Comment by mdreizin** [comment]
@b1551 Could you please disable GitHub Actions on your fork?

**Comment by mdreizin** [comment]
@b1551 any updates?

**Comment by b1551** [comment]
disabled them just now

**Pull request merged**